src/main/java/org/mandas/docker/client/DockerHost.java

```diff
@@ -243,14 +242,14 @@ static String configPathFromEnv() {
 
   @Override
   public String toString() {
-    return MoreObjects.toStringHelper(this)
-        .add("host", host)
-        .add("uri", uri)
-        .add("bindUri", bindUri)
-        .add("address", address)
-        .add("port", port)
-        .add("certPath", certPath)
-        .toString();
+	return "DockerHost{"
+		+ "host=" + host
+		+ ",uri=" + uri
+		+ ",bindUri=" + bindUri
+		+ ",address=" + address
+		+ ",port=" + port
+		+ ",certPath=" + certPath == null? "null": certPath
```

Comment:
if you decide to check for existence above, you should check for existence here also